scripts/Set-NetAdapterPriorityVLanTag.ps1

```diff
@@ -0,0 +1,52 @@
+function Set-NetAdapterPriorityVLanTag
```

Comment:
Can you add a comment at the very top on why we need this script.

Reply:
Yessir